virttest/env_process.py

```diff
@@ -1047,6 +1050,11 @@ def preprocess(test, params, env):
                     env["cpu_driver"] = cpu_driver[0]
                     params["cpu_driver"] = env.get("cpu_driver")
 
+    kvm_module_params = params.get("kvm_module_parameters", "")
+    force_load = params.get("kvm_module_force_load", "no") == "yes"
+    global KVM_MODULE_HANDLER
+    KVM_MODULE_HANDLER = utils_kernel_module.reload("kvm", force_load, kvm_module_params)
```

Comment:
Too long line

Reply:
reload is indeed better name, but on the other hand it might give false impression we are always reloading the module. It's rather `reload_if_needed` (but that is a terrible and long name). Anyway I don't have a better suggestion here so let's just leave it as is.

Reply:
@ldoktor I understand your suggestions but I agree it's a very long name and also the 'force_load' somehow implies that there are situations to force the action, maybe.